modules/home-environment.nix

```
@@ -508,6 +508,8 @@ in
       if config.submoduleSupport.enable
         && config.submoduleSupport.externalPackageInstall
       then "/etc/profiles/per-user/${cfg.username}"
+      else if config.nix.settings.use-xdg-base-directories
```

I think you'll need something like

```suggestion
      else if config.nix.enable && (config.nix.settings.use-xdg-base-directories or false)
```

---

Done